src/utils/data.ts

```diff
@@ -149,3 +149,42 @@ export function maxInIterator<T>(iterator: Iterator<T>, getItemScore: (item: T)
 
   return maxItem
 }
+
+function encode(text: string): Uint8Array {
+  // Benchmark: https://jsbench.me/b6klaaxgwq/1
+  // If you want to just count bytes, see solutions at https://jsbench.me/ehklab415e/1
+  if (typeof TextEncoder === 'function') {
+    return new TextEncoder().encode(text) // From https://stackoverflow.com/a/11411402/1118709
+  }
+
+  // From https://stackoverflow.com/a/18722848/1118709
+  const binaryText = unescape(encodeURI(text))
```

Comment:
The fallback is not necessary, all browsers that we support (and some older browsers) do support `TextEncoder`